docs/book/tutorials/kubernetes-on-vsphere-with-kubeadm.md

```diff
@@ -225,6 +229,26 @@ Server Version: 18.06.0-ce
 Cgroup Driver: systemd
 ```
 
+### VMTools NIC/Device Filtering
+
+A number of [CNI](https://github.com/containernetworking/cni) implementations (such Calico, Antrea, and etc) introduce networking artifacts that interfere with the normal operation of vSphere's internal reporting for network/device interfaces. To address this issue, an `exclude-nics` filter for VMTools needs to be applied in order to prevent these artifacts from getting reported to vSphere and causing problems with network/device associations to vNICs on virtual machines.
+
+The recommended `exclude-nics` filter is as follows for `/etc/vmware-tools/tools.conf`:
+
+```bash
+[guestinfo]
+primary-nics=eth0
+exclude-nics=antrea-*,ovs-system,br*,flannel*,veth*,docker*,virbr*,vxlan_sys_*,genev_sys_*,gre_sys_*,stt_sys_*,????????-??????
```

Comment:
I think the `????????-??????` filter is gonna be confusing to a lot of users, should we add more words explaining this?

Reply:
Good point